pkg/apis/operator/v1alpha1/addon_types.go

```diff
@@ -38,7 +38,7 @@ type AddonCondition struct {
 
 // Addon is the Schema for the addons API
 // +k8s:openapi-gen=true
-// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=addons,scope=Namespaced
```

Comment:
aren't the addons cluster-scoped?

Reply:
Sorry this should be reverted, it was an attempt to fix the bug that did not do what I expected.